conda/recipes/versions.yaml

```diff
@@ -2,7 +2,7 @@
 
 # Versions for `rapids-xgboost` meta-pkg
 xgboost_version:
-  - '=2.1.4'
```

Comment:
Follow [the package match specifications]( https://docs.conda.io/projects/conda-build/en/stable/resources/package-spec.html#package-match-specifications ), this uses an `|` to allow either the XGBoost 2 or 3 version that we will supply in this release